src/org/python/core/PyReflectedFunction.java

```diff
@@ -162,13 +162,26 @@ protected void addArgs(ReflectedArgs args) {
     public PyObject __call__(PyObject self, PyObject[] args, String[] keywords) {
         ReflectedCallData callData = new ReflectedCallData();
         ReflectedArgs match = null;
+        // varargs methods should always have lower precedence than a non-varargs
+        // method that also applies but as they will always have <= arguments than an
```

Comment:
I'm not sure this is true but I don't think it matters. eg

```python
foo(1,2)
```
would match both
```java
public void foo(int i, int j) {}
public void foo(int i, int j, int... k) {}
```
The vararg case has more arguments but still shouldn't be used. As (I think) the methods are in arg count order, the vararg method will be later and therefore never checked.